src/SFA.DAS.ApplyService.InternalApi/Infrastructure/ApiClientBase.cs

```diff
@@ -0,0 +1,232 @@
+﻿using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using System;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.ApplyService.InternalApi.Infrastructure
+{
+    public abstract class ApiClientBase<CB>
```

Comment:
This class should be reused, it seems there are 2 copies of it?

Reply:
I can try to figure out if we can shove this somewhere else. Seems overkill to have a new class library for it. I'd imagine sticking it anywhere else due to it being Infrastructure would create problems though.

Reply:
If we end up with a new project I'm sure there are other pieces of duplicate code which could go in it.

Reply:
Done